utils/http/transport.go

```diff
@@ -1,58 +0,0 @@
-package http
-
-import (
-	"net"
-	"net/http"
-	"runtime"
-	"time"
-)
-
-// code source: https://github.com/hashicorp/go-cleanhttp/blob/master/cleanhttp.go
-// DefaultTransport returns a new http.Transport with similar default values to
-// http.DefaultTransport, but with idle connections and keepalives disabled.
-func DefaultTransport() *http.Transport {
```

Comment:
why are we removing these functions?

Reply:
not removing, just move into `dce-plugin`

Reply:
done, reverted the change